react/src/utils/error-handling.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+import { useToast, UseToastOptions } from "@chakra-ui/react";
+import { useCallback } from "react";
+
+export interface ITriggerErrorProps extends UseToastOptions {
+  message: string;
+  userMessage?: string;
+  statusCode?: string;
+}
+
+export const useErrorHandling = () => {
+  const toast = useToast();
+
+  const triggerError = useCallback(
+    ({ message, userMessage, statusCode }: ITriggerErrorProps) => {
+      if (statusCode) console.error(`[${statusCode}] ${message}`);
+      else console.error(`${message}`);
+
+      toast({
+        duration: 5000,
+        isClosable: true,
+        position: "top",
+        variant: "subtle",
+        status: "error",
+        title: "Error",
```

Comment:
Should we also make the toast title customizable as before, e.g. for displaying the box label identifier?

Reply:
Clarified: customizable through `createToast` hook